include/command.h

```diff
@@ -269,6 +292,16 @@ class Command
         */
         std::vector<char*>_local_fields;
 
+        /*!
+        *   \brief Address of database node
+        */
+        std::string _address;
+
+        /*!
+        *   \brief Port of database node
+        */
+        uint64_t _port;
```

Comment:
I think we will want to specify a default value of ``_port`` that is something that can be distinguished as ``Command`` does not have an address and port for execution.  Reading more about standards, it seems like ports are constrained to values that fit into a 16-bit unsigned integer (maybe we change the data type throughout the code to reflect this?).

Reply:
Ok i will change the `_port` to be a `uint16_t`. The default for an `unit16_t` is `0`. Do you think there would be any case where the `_port` would be set to `0`? If not, we can keep that as the default.